build-dependencies.sh

```diff
@@ -18,23 +18,22 @@ make install
 
 # Install cryptography and yara-python
 cd /
+pip3 install --upgrade pip
 mkdir pip
-pip3 install cryptography yara-python -t pip
+/usr/local/bin/pip3 install cryptography yara-python -t pip
```

Comment:
Can you remind me (and maybe add a comment explaining) why these need to be installed here rather than in requirements.txt with the other Pythong depedencies?

Reply:
These are the runtime dependencies, the python packages along with some bits installed from the yum repositories and packaged up into a zip which allow yara to run. 
The dependencies in `requirements.txt` are dependencies needed to run the tests. We only install them just before we run the tests.

Reply:
Shouldn't we be using requirements.txt for the runtime environment as well? Otherwise it's a bit misleading - the version pinned in requirements.txt will satisfy Snyk, but it won't be the version that's actually installed.